cloudinit/config/tests/test_ubuntu_drivers.py

```diff
@@ -16,6 +16,13 @@
     "(choose from 'list', 'autoinstall', 'devices', 'debug')\n")
 
 
+# The tests in this module call helper methods which are decorated with
+# mock.patch.  pylint doesn't understand that mock.patch passes in parameters
```

Comment:
I think instead of `passes in parameters` should be `passes its parameters` or something like that

Reply:
I think it should actually be "passes parameters" (so the whole fragment is "mock.patch passes parameters to the decorated function"), let me fix that.